tests/test_maskStreaks.py

```diff
@@ -0,0 +1,142 @@
+# This file is part of pipe_tasks.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+import numpy as np
+
+import lsst.utils.tests
+import lsst.afw.image
+import lsst.afw.math
+from lsst.pipe.tasks.maskStreaks import MaskStreaksTask, LineProfile, Line, setDetectionMask
+
+
+class TestMaskStreaks(lsst.utils.tests.TestCase):
+
+    def setUp(self):
+        self.config = MaskStreaksTask.ConfigClass()
+        self.config.dChi2Tolerance = 1e-6
+        self.fst = MaskStreaksTask(config=self.config)
+
+        self.testx = 500
+        self.testy = 600
+        self.exposure = lsst.afw.image.ExposureF(self.testy, self.testx)
+        rand = lsst.afw.math.Random()
+        lsst.afw.math.randomGaussianImage(self.exposure.image, rand)
+        self.exposure.maskedImage.variance.set(1)
+        self.maskName = "STREAK"
+        self.detectedPlane = "DETECTED"
+
+    """def test_input(self):
```

Comment:
Something is up with this test, it is commented out? Should you put it back in? Should it be taken out? As an fyi you can use `unittest.skip` decorator to skip a test while you are testing and debugging. The final test output will indicate that one test was skipped so it is not forgotten like a comment.

Reply:
No, this test should have been removed. It is no longer relevant, since we don't allow a separate "binary image" anymore.